src/components/views/rooms/RoomPreviewBar.js

```diff
@@ -31,6 +48,8 @@ module.exports = React.createClass({
         onJoinClick: PropTypes.func,
         onRejectClick: PropTypes.func,
         onForgetClick: PropTypes.func,
+        onSignInClick: PropTypes.func,
+        onSignUpClick: PropTypes.func,
```

Comment:
Should we add `loading`, `busy`, `room_alias`, etc. to the `propTypes` somewhere? It seems like there are quite a few unchecked props.

Reply:
Yes, forgot about these! Ftr: room_alias was a typo, and busy is state, added the others!